jps-plugin/src/com/goide/jps/model/JpsGoSdkType.java

```diff
@@ -43,4 +43,9 @@ public JpsDummyElement createDefaultProperties() {
   private static File getExecutable(@NotNull String path, @NotNull String command) {
     return new File(path, getBinaryFileNameForPath(command));
   }
+
+  @NotNull
+  public static File getExecutableFileFromEnv(String command) {
```

Comment:
You are right. It would crash, if goimports is not install. I could add a notification if the method returns null.